src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -482,6 +477,8 @@ public List<PackageManagerSearchElement> ListAll()
         {
             CachedPackageList = new List<PackageManagerSearchElement>();
 
+            // Calls to Model.UserVotes and Model.ListAll might take a long time to run (so do not use them syncronously in the UI thread)
```

Comment:
public List<PackageManagerSearchElement> ListAll() is called from an async method upstream
It is not going to block the UI thread anymore.

We do need a clear way to mark these methods (that they are Task based or async) . This will be part of another task

Reply:
what async method calls `ListAll()`?

Reply:
https://github.com/DynamoDS/Dynamo/blob/d01e0572aec56fbb142e85495bb2a6a6ba569871/src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs#L1026